ansible_runner/utils.py

```diff
@@ -326,6 +306,19 @@ def _emit_event(self, buffered_stdout, next_event_data=None):
             event_data['end_line'] = self._start_line + n_lines
             self._start_line += n_lines
             if self._event_callback:
+                if event_data.get('stdout'):
```

Comment:
Even *if* this PR doesn't work out for replacing partials we should probably still do this. Trying to emit stdout down at the callback plugin gives us all kinds of grief.

Reply:
Agreed, this is a much saner place to handle `sys.stdout`